osu.Game/Graphics/UserInterface/FixedSearchTextBox.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class FixedSearchTextBox : SearchTextBox
```

Comment:
What does "fixed" mean here?

Reply:
The caret is "fixed". Sounds misleading now, but wasn't too sure of what to name it in the first place.

Reply:
`SeekLimitedSearchTextBox` maybe... hard to come up with a good name